packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -465,53 +494,48 @@ const publicBehavior = {
     /** @type {VaultManager} */
     return collateralTypes.get(brandIn).getPublicFacet();
   },
-  /**
-   * @param {MethodContext} context
-   */
-  getMetrics: ({ state }) => state.metricsSubscription,
+  getMetrics: () => ephemera.metricsSubscription,
 
   /** @deprecated use getCollateralManager and then makeVaultInvitation instead */
   makeLoanInvitation: makeVaultInvitation,
   /** @deprecated use getCollateralManager and then makeVaultInvitation instead */
   makeVaultInvitation,
   getCollaterals,
-  /** @param {MethodContext} context */
-  getRunIssuer: ({ state }) => state.debtMint.getIssuerRecord().issuer,
+  getRunIssuer: () => ephemera.debtMint.getIssuerRecord().issuer,
   /**
    * subscription for the paramManager for a particular vaultManager
    *
    * @param {MethodContext} context
    * @param {{ collateralBrand: Brand }} selector
    */
-  getSubscription: ({ state }, { collateralBrand }) =>
-    state.vaultParamManagers.get(collateralBrand).getSubscription(),
+  getSubscription: (context, { collateralBrand }) =>
+    ephemera.vaultParamManagers.get(collateralBrand).getSubscription(),
   /**
    * subscription for the paramManager for the vaultFactory's electorate
-   *
-   * @param {MethodContext} context
    */
-  getElectorateSubscription: ({ state }) =>
-    state.directorParamManager.getSubscription(),
+  getElectorateSubscription: () =>
+    ephemera.directorParamManager.getSubscription(),
   /**
    * @param {MethodContext} context
    * @param {{ collateralBrand: Brand }} selector
    */
-  getGovernedParams: ({ state }, { collateralBrand }) =>
+  getGovernedParams: (context, { collateralBrand }) =>
```

Comment:
Why isn't lint complaining about these?

```suggestion
  getGovernedParams: (_context, { collateralBrand }) =>
```

Reply:
Because it's a param list and the value after is is used.